cypress/e2e/Assistant.spec.js

```diff
@@ -101,13 +101,16 @@ describe('Assistant', () => {
 			.should('be.visible')
 			.click()
 
+		// eslint-disable-next-line cypress/no-unnecessary-waiting
+		cy.wait(1000)
```

Comment:
I did not figure out a way to make this wort otherwise. For some reason running on CI otherwise focusses the NcSelect 🤷